### DISCUSSION TITLE
DBeaver comunity edition - auto data refresh

### DISCUSSION THREAD

**Discussion by PalmaPedro** [dbeaver/dbeaver#17889] [General]
Hello, 

I'm a new dbeaver user and need some help. I searched the forum but could not find any mention of this, I would like to set auto refresh for my database tables, I found this in the documentation : 

https://dbeaver.com/docs/wiki/Data-Refresh/

I can't seem to find this feature in the community edition, is it not available?

thank you

**Comment by LonwoLonwo** [comment]
Hello  @PalmaPedro 

You can find the Refresh button in the Data Viewer right corner.

![2022-10-12 19_09_28-DBeaver Ultimate 22 2 2 - test_DIFFERENT_case](https://user-images.githubusercontent.com/45152336/195394052-4de6d4f8-ead7-4821-a8fb-811c9ce316a9.png)

**Reply by PalmaPedro** [reply]
Hi, thank you :)
